README.md

```diff
@@ -43,13 +43,63 @@ Finally, generate an invite link for the bot and enter it into any browser.
 > [!NOTE]
 > Bot permissions involving threads/mentions/messages should suffice, but you can naively give it `Administrator` since it's just a test bot in your own testing Discord server.  
 
+### Database
+
+The leaderboard persists information in a Postgres database. To develop locally, set Postgres up on your machine. Then create a database with:
+
+```
+CREATE DATABASE clusterdev;
```

Comment:
where do i type this in? just shell?

Reply:
In the Postgres shell. I'll edit the readme to include more details.